### ISSUE TITLE
Support for Notification Subscriptions

### ISSUE THREAD

**Issue by alexanderlinne** [microsoft/terraform-provider-azuredevops#907]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

The Azure DevOps REST API supports settings up notification subscriptions. This does not currently seem to be supported by this provider.

### New or Affected Resource(s)

* azuredevops_XXXXX

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000

**Comment by xuzhang3** [comment]
@alexanderlinne Do you mean `Settings -> Notifications -> New subscription` ?
![image](https://github.com/microsoft/terraform-provider-azuredevops/assets/57888764/081db6cc-2bc8-4477-8cec-166a99ed39b5)

**Comment by alexanderlinne** [comment]
> @alexanderlinne Do you mean `Settings -> Notifications -> New subscription` ?

Yes, and I would expect that you could also include the Delivery Settings. This would enable us to automatically set up and configure alerting for Pipelines deployed using Terraform

**Comment by fsdrw08** [comment]
any updates?
